lib/win32/certstore/certstore.rb

```diff
@@ -15,30 +15,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-require "win32/certstore/version"
-require 'win32/api/reserved_names'
-
 module Win32
-  module Certstore
-    include Win32::API::ReservedNames
-    include Chef::Mixin::WideString
+  class Certstore
+    # CA -> Certification authority certificates.
+    # MY -> A certificate store that holds certificates with associated private keys.
+    # ROOT -> Root certificates.
+    # SPC -> Software Publisher Certificate.
 
-    def open store_name

```

Comment:
We should have `open` and `close` methods in this file for `Certstore` instead of `win32_base` file, probably with different names if `open` and `close` are too general.